src/components/Identity/processIdSyncs.js

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { assign, cookie, fireDestinations } from "../../utils";
+import namespace from "../../constants/namespace";
+
+const millisecondsPerHour = 60 * 60 * 1000;
+
+const getControlObject = () => {
+  return JSON.parse(cookie.get(`${namespace}idSyncControl`) || "{}");
+};
+
+const setControlObject = obj => {
+  cookie.set(`${namespace}idSyncControl`, JSON.stringify(obj), {
```

Comment:
Let's try this format so we can save more space:

```
{
    syncs: ID-TS_ID-TS_ID-TS
}
```
Since `encodeURIComponent("-_")` outputs `-_`.